File: Cap xA/Capitulo 03 - Exercicio 4a/c03ex4a.java
/*
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: Apendice A - CORRECAO DE ALGUNS EXERCICIOS DE FIXACAO
 * 
 * Programa ..: CONVERSAO DE TEMPERATURA
 */

import java.util.Scanner;

public class Main {
	
  static double C, F;
  
  public static void main(String[] args) {
	  
    System.out.println("CONVERSAO DE TEMPERATURA");
    System.out.println();
    System.out.print("Entre temperatura em Celsius ..: ");
    Scanner entrada = new Scanner(System.in);
    C = entrada.nextDouble();
    F = C * 9 / 5 + 32;
    System.out.printf("Em Fahrenheit equivale a ......: %.1f%n", F);
    entrada.close();
    
  }
  
}

File: Cap xA/Capitulo 03 - Exercicio 4a/c03ex4a.kt
/*
 * Livro .....: Algoritmos
 *              Lógica para Desenvolvimento de Programação de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: Apendice A - CORRECAO DE ALGUNS EXERCICIOS DE FIXACAO
 *
 * Programa ..: CONVERSAO DE TEMPERATURA
 */

import java.util.Scanner

fun main() {
  
    var entrada = Scanner(System.`in`)

    println("CONVERSAO DE TEMPERATURA")
    println()

    print("Entre temperatura em Celsius ..: ")
    var C = entrada.nextFloat()

    var F = C * 9 / 5 + 32

    println("Em Fahrenheit equivare a ......: $F")
    
}
